src/main/java/com/atlassian/db/replica/api/Database.java

```diff
@@ -0,0 +1,8 @@
+package com.atlassian.db.replica.api;
+
+import java.sql.Connection;
+
+public interface Database {
```

Comment:
How about adding method: `getRole() `
`Reader`, `Writer`

Reply:
Please explain the use case.